Final_project/sql/enrich_user_profiles.sql

```diff
@@ -0,0 +1,31 @@
+DELETE FROM `{{ params.project_id }}.3_gold.user_profiles_enriched`
```

Comment:
I wrote in another comment that you should use either DELETE or MERGE, not both at the same time.